crates/bevy_sprite/src/render/sprite.wgsl

```diff
@@ -6,8 +6,8 @@ struct View {
     projection: mat4x4<f32>,
     inverse_projection: mat4x4<f32>,
     world_position: vec3<f32>,
-    width: f32,
-    height: f32,
+    // viewport(x_origin, y_origin, width, height)
+    viewport: vec4<f32>,
```

Comment:
could you import from `bevy_sprite::mesh2d_view_types` instead of redefining it here?

Reply:
Good idea, I think

Reply:
Or maybe it should really be a shader imports defined in bevy_render and imported everywhere else…

Reply:
this is done in #5535